### DISCUSSION TITLE
Toast 컴포넌트 관련 질문

### DISCUSSION THREAD

**Discussion by hyoribogo** [prgrms-fe-devcourse/FEDC4_HONKOK_JunilHwang#34] [Q&A]
1. `Toast` 컴포넌트를 강의대로 만들려면 `useTimeout` 훅과 `Text` 컴포넌트를 따로 만들어야 하고(안 만들어도 되겠지만?), `uuid` 라이브러리의 `v4` 함수를 사용해야 합니다.
2. `Toast` 컴포넌트를 클래스 컴포넌트로 만들어서 스토리북 component에 할당이 되지 않습니다.
3. 타입스크립트 부분이 좀 어렵습니다 😅

이 문제들을 같이 얘기해보고 싶습니다!

**Comment by jgjgill** [comment]
강의 기반으로 코드를 작성한 것일까요?
강의와 코드가 일치하면 다른 부분은 타입스크립트를 사용한 것과 스토리북 설정일 것 같은데 어떤 에러가 발생하는지 확인해 볼 필요가 있겠네요..
원격저장소에 커밋 올리시면 지금 한 번 저도 확인해보겠습니다..

**Reply by hyoribogo** [reply]
일단 커밋 올렸습니다 ! 😂
